openapi/index.yaml

```diff
@@ -174,14 +175,33 @@ definitions:
       status: ACTIVATED
       activation_date: "2021-10-13T00:00:00.000Z"
       expiration_date: "2030-10-13T00:00:00.000Z"
-  CgnRevocationRequest:
+  CgnStatusActivationRequest:
     type: object
     properties:
+      action:
+        type: string
+        x-extensible-enum:
+          - ACTIVATE
+    required:
+      - action
+  CgnStatusRevocationRequest:
+    type: object
+    properties:
+      action:
+        type: string
+        x-extensible-enum:
+          - REVOKE
       revocation_reason:
```

Comment:
Can we rename this prop with a more generic `reason`? This could be used for all status update requests

Reply:
Nope the `reason` attribute is specifically defined for `REVOKED` CGNs. :)